Podfile

```diff
@@ -28,6 +28,7 @@ abstract_target "App" do
     pod 'SwiftUIIntrospect', '~> 0.8'
     pod 'Kingfisher', '~> 7.8'
     pod 'Swinject', '2.8.3'
+    pod 'BranchSDK', '~> 3.0.1'
```

Comment:
Why do we using pod if there is Swift Package for Branch (https://github.com/BranchMetrics/ios-branch-sdk-spm)?
I thought we were trying to get away from using pods

Reply:
I wasn't aware of that, added as swift package.